wsd/LOOLWSD.cpp

```diff
@@ -1174,21 +1174,12 @@ void LOOLWSD::initialize(Application& self)
 
 #if ENABLE_SSL
     LOOLWSD::SSLEnabled.set(getConfigValue<bool>(conf, "ssl.enable", true));
-#endif
-
-    if (LOOLWSD::isSSLEnabled())
-    {
-        LOG_INF("SSL support: SSL is enabled.");
-    }
-    else
-    {
-        LOG_WRN("SSL support: SSL is disabled.");
-    }
-
-#if ENABLE_SSL
     LOOLWSD::SSLTermination.set(getConfigValue<bool>(conf, "ssl.termination", true));
 #endif
 
+    LOG_INF("SSL support: SSL is " << (LOOLWSD::isSSLEnabled() ? "enabled." : "disabled."));
+    LOG_INF("SSL support: termination is " << (LOOLWSD::isSSLTermination() ? "enabled." : "disabled."));
```

Comment:
I wonder if it makes sense to explain the termination with a few words actually ;-)

Something like "SSL support: termination (for a SSL proxy) is " << ...?

Either way, thanks a lot for improving this!

Reply:
Oh sorry @kendy, I completely missed this comment. Should I amend the log text in a separate commit, then?

Reply:
@thebearon : No problem at all, a separate commit is completely OK of course :-)